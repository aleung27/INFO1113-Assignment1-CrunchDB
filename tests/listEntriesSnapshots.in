LIST ENTRIES
SET . 3
SNAPSHOT
LIST ENTRIES
SET [ 0
LIST ENTRIES
SNAPSHOT
CHECKOUT 1
LIST ENTRIES
SET k 9
LIST ENTRIES
CHECKOUT 1
LIST ENTRIES
ROLLBACK 2
LIST ENTRIES
