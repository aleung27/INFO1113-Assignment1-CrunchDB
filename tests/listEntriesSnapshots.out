> no entries

> ok

> saved as snapshot 1

> . [3]

> ok

> [ [0]
. [3]

> saved as snapshot 2

> ok

> . [3]

> ok

> k [9]
. [3]

> ok

> . [3]

> ok

> [ [0]
. [3]

